plugins/woocommerce/includes/wc-conditional-functions.php

```diff
@@ -256,18 +256,6 @@ function is_lost_password_page() {
 	}
 }
 
-if ( ! function_exists( 'is_ajax' ) ) {
-
-	/**
-	 * Is_ajax - Returns true when the page is loaded via ajax.
-	 *
-	 * @return bool
-	 */
-	function is_ajax() {
-		return function_exists( 'wp_doing_ajax' ) ? wp_doing_ajax() : Constants::is_defined( 'DOING_AJAX' );
-	}
-}
```

Comment:
We can't simply remove a public function in this fashion. It needs to be deprecated first of all (otherwise, we risk show-stopping breakages when third party code that has not yet been updated breaks).

Reply:
But I did deprecate it? See includes/wc-deprecated-functions.php in this PR

Reply:
Yes you did :+1: